scheduler/scheduler_test.go

```diff
@@ -1,26 +1,35 @@
 package scheduler
 
 import (
+	"crypto/tls"
 	"hash/fnv"
 	"os"
 	"os/exec"
 	"path/filepath"
 	"testing"
 
 	"github.com/gaia-pipeline/gaia"
+	"github.com/gaia-pipeline/gaia/security"
 	"github.com/gaia-pipeline/gaia/store"
 	hclog "github.com/hashicorp/go-hclog"
 	uuid "github.com/satori/go.uuid"
 )
 
 type PluginFake struct{}
 
-func (p *PluginFake) NewPlugin() Plugin                            { return &PluginFake{} }
+func (p *PluginFake) NewPlugin(ca security.CAAPI) Plugin           { return &PluginFake{} }
 func (p *PluginFake) Connect(cmd *exec.Cmd, logPath *string) error { return nil }
 func (p *PluginFake) Execute(j *gaia.Job) error                    { return nil }
 func (p *PluginFake) GetJobs() ([]gaia.Job, error)                 { return prepareJobs(), nil }
 func (p *PluginFake) Close()                                       {}
 
+type CAFake struct{}
+
+func (c *CAFake) CreateSignedCert() (string, string, error)                       { return "", "", nil }
```

Comment:
Okay, so I know that this will be annoying, but... :D Could you please change the API of the CA to named values in the return statement? Because it's annoying that I don't know what `"", "", nil` is actually returning by looking at the pop up of the parameters. :D I don't which is which here. Path, or Cert or KeyPath or whatever, you see what I'm saying?